src/cpp/lib/QtGui/QFont/qfont_wrap.cpp

```diff
@@ -0,0 +1,168 @@
+}
+
+Napi::Value QFontWrap::setFamily(const Napi::CallbackInfo& info) {
+  Napi::Env env = info.Env();
+  Napi::HandleScope scope(env);
+  std::string family = info[0].As<Napi::String>().Utf8Value();
+  this->instance->setFamily(QString::fromStdString(family.c_str()));
+  return env.Null();
+}
+
+Napi::Value QFontWrap::family(const Napi::CallbackInfo& info) {
+  Napi::Env env = info.Env();
+  Napi::HandleScope scope(env);
+  QString family = this->instance->family();
+  return Napi::String::New(env, family.toStdString());
+}
+
+Napi::Value QFontWrap::setPointSize(const Napi::CallbackInfo& info) {
+  Napi::Env env = info.Env();
+  Napi::HandleScope scope(env);
+  int pointSize = info[0].As<Napi::Number>().Int32Value();
+  this->instance->setPointSize(pointSize);
+  return env.Null();
+}
+
+Napi::Value QFontWrap::pointSize(const Napi::CallbackInfo& info) {
+  Napi::Env env = info.Env();
+  Napi::HandleScope scope(env);
+  return Napi::Value::From(env, this->instance->pointSize());
+}
+
+Napi::Value QFontWrap::setStretch(const Napi::CallbackInfo& info) {
+  Napi::Env env = info.Env();
+  Napi::HandleScope scope(env);
+  int factor = info[0].As<Napi::Number>().Int32Value();
+  this->instance->setStretch(factor);
+  return env.Null();
+}
+
+Napi::Value QFontWrap::stretch(const Napi::CallbackInfo& info) {
+  Napi::Env env = info.Env();
+  Napi::HandleScope scope(env);
+  return Napi::Value::From(env, this->instance->stretch());
+}
+
+Napi::Value QFontWrap::setWeight(const Napi::CallbackInfo& info) {
+  Napi::Env env = info.Env();
+  Napi::HandleScope scope(env);
+  int weight = info[0].As<Napi::Number>().Int32Value();
+  this->instance->setWeight(weight);
+  return env.Null();
+}
+
+Napi::Value QFontWrap::weight(const Napi::CallbackInfo& info) {
+  Napi::Env env = info.Env();
+  Napi::HandleScope scope(env);
+  return Napi::Value::From(env, this->instance->weight());
+}
+
+Napi::Value QFontWrap::setItalic(const Napi::CallbackInfo& info) {
+  Napi::Env env = info.Env();
+  Napi::HandleScope scope(env);
+  int enable = info[0].As<Napi::Boolean>().Value();
+  this->instance->setItalic(enable);
+  return env.Null();
+}
+
+Napi::Value QFontWrap::italic(const Napi::CallbackInfo& info) {
+  Napi::Env env = info.Env();
+  Napi::HandleScope scope(env);
+  return Napi::Value::From(env, this->instance->italic());
+}
+
+Napi::Value QFontWrap::toString(const Napi::CallbackInfo& info) {
+  Napi::Env env = info.Env();
+  Napi::HandleScope scope(env);
+  QString value = this->instance->toString();
+  return Napi::String::New(env, value.toStdString());
+}
+
+Napi::Value StaticQFontWrapMethods::fromQVariant(
+    const Napi::CallbackInfo& info) {
+  Napi::Env env = info.Env();
+  Napi::HandleScope scope(env);
+  Napi::Object variantObject = info[0].As<Napi::Object>();
+  QVariantWrap* variantWrap =
+      Napi::ObjectWrap<QVariantWrap>::Unwrap(variantObject);
+  QVariant* variant = variantWrap->getInternalInstance();
+  QFont color = variant->value<QFont>();
```

Comment:
tiny knit pick - can you change the color to value or font

Reply:
alright, done!